### DISCUSSION TITLE
Welcome to AISH Discussions!

### DISCUSSION THREAD

**Discussion by SteveL-MSFT** [PowerShell/AIShell#158] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anshsinghal2002** [comment]
Hi, is this an open source project seeking collaboration?
